lib/forcex/client.ex

```diff
@@ -61,6 +61,20 @@ defmodule Forcex.Client do
     services = Forcex.services(client)
     %{client | services: services}
   end
+  
+  def create_sobject(client \\ %__MODULE__{}, name \\ "SOBject", map \\ %{})
+
+  def create_sobject(client, name, map) when is_atom(name) do
+    name = name 
+    |> Atom.to_string
+    |> String.capitalize
+
+    client
+    |> create_sobject(name, map)
+  end
+  def create_sobject(client, name, map) do
+    Forcex.post("/services/data/v20.0/sobjects/#{name}", map, client)
```

Comment:
@jeffweiss curious why you chose to have Forcex module methods use a trailing client param rather than leading (for pipeline operator)